### PR TITLE
Fixed username sorting of tagged users in dashboard

### PR DESCRIPTION
**Pull request by githue**
* sorting tagged users by username now works even after re-sorting other columns.
* added 'Page' before pagination buttons, and fixed their positioning.
* tableTools no longer affects dashboard. Was requiring an extra click to sort by username.

### CODE REVIEW

File: lib/modules/userTagger.js [hunk 1]
```diff
@@ -506,14 +507,11 @@ modules['userTagger'] = {
...
+				break;
 			default:
-				// sort users, ignoring case
-				taggedUsers.sort(function(a, b) {
-					return (a.toLowerCase() > b.toLowerCase()) ? 1 : (b.toLowerCase() > a.toLowerCase()) ? -1 : 0;
-				});
-				if (this.descending) {
-					taggedUsers.reverse();
-				}
+				modules['userTagger'].sortUsername(taggedUsers, this.decending);
```
Comment (andytuba) [inline lib/modules/userTagger.js]: typo -- `descending`

File: lib/modules/userTagger.js [hunk 2]
```diff
@@ -506,14 +507,11 @@ modules['userTagger'] = {
 					taggedUsers.reverse();
 				}
 				break;
+			case 'username':
+				modules['userTagger'].sortUsername(taggedUsers, this.descending);
```
Comment (andytuba) [inline lib/modules/userTagger.js]: may as well  simply `case 'username': // falls through` and delete the rest of this case.

Comment (githue) [inline lib/modules/userTagger.js]: JSHint gives a warning when I do that.

> `Expected a 'break' statement before 'default'.`

Comment (andytuba) [inline lib/modules/userTagger.js]: Oh, sorry, JSHint is expecting this:

    case 'username':
        /* falls through */
    default:

**Pull request merged**